articles/20241126_Using_Environmental_Variables_in_Daytona.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Using Environmental Variables in Daytona'
+description: 'Learn how to set, manage, and use environmental variables in Daytona workspaces with a simple Python project for demonstration.'
+date: 2024-11-26
+author: 'Busayo Samuel'
+tags: ['Environment Variables', 'Development Environment', 'Daytona Workspaces']
+---
+
+# Using Environmental Variables in Daytona
+
+## Introduction
+
+[Environmental variables]('/definitions/20241126_definition_environment_variables.md') are important for configuring development environments and storing sensitive information. They provide a flexible way to control application behavior without hardcoding values directly into the source code. Daytona offers a streamlined method for setting, managing, and using environmental variables across various workspaces using the `daytona env` command. This simplifies configuration and enhances security especially when collaborating on projects.
+
+This article will explain using the `daytona env` command to set and manage environmental variables within Daytona workspaces. You will learn how to set variables, create a workspace, and access these variables within a workspace.
+
+### TL;DR
+
+- **What**: Learn to use `daytona env set` for managing workspace environmental variables
+
+- **Why**: Simplify configuration and securely manage development environment settings
+
+- **How**:
+  - Set variables using `daytona env set key=value`
+  - Create a workspace with `daytona create --code`
+  - Verify variables with `daytona env list`
+
+## Prerequisites
+
+- Daytona installed on your system
+- Docker
+- Visual Studio Code (VSCode)
+
+## Set Environmental Variables
+
+Before adding an environment variable to Daytona, ensure that Docker and your server are running. Start your server using:
+
+```bash
+daytona server
+```
+
+Then use `daytona env set` to define environmental variables:
```

Comment:
mention they can use TUI, editor or in a cli like in your example, add one more screenshot of TUI